### DISCUSSION TITLE
Change of license

### DISCUSSION THREAD

**Discussion by theogiraudet** [Gunivers/Bookshelf#184] [General]
Hello there!

Currently, Bookshelf (formerly Gunivers-Lib) is licensed under the CC-BY-NC-SA 4.0 license. Although this is an open-source license, it prohibits commercial use. 
This may discourage people from using Bookshelf in their projects. Always with the idea of opening Bookshelf to as many people as possible with as few restrictions as possible, we propose to change the license of Bookshelf to MPL 2.0 (Mozilla Public License 2.0).

Here are the main points of the MPL 2.0 license:
- Bookshelf may be redistributed and modified by others.
- The license of the entire content of Bookshelf should keep the MPL 2.0 license or adopt a compatible license, even if the content is modified.
- A project using Bookshelf can adopt any license (without changing the license of the Bookshelf files).
- If some files of Bookshelf are modified, the source code of these modified files should be disclosed.
- We (Gunivers & the contributors) retain our attribution.
- Commercial use is allowed.
More information about the licence [here](https://choosealicense.com/licenses/mpl-2.0/).

To adopt this new license, we need the authorization of the different contributors of the project (that still have code in the current state of the project).
@Leirof (✅), @aksiome (✅), @KubbyDev (✅), @Aeris1One (✅), @ascpial (✅), @Ethanout (✅), @Majoras16 (✅), @theogiraudet (✅), please can you write "I accept this change of license for the entire Bookshelf project." as a comment on this discussion?
If you disagree (or do not answer), we'll see about removing your work from the project to respect your choice.
If you'd like to discuss this choice, please don't hesitate!

Thanks!

**Comment by theogiraudet** [comment]
Of course, I accept this change of license for the entire Bookshelf project. :)

**Comment by aksiome** [comment]
I accept this change of license for the entire Bookshelf project.

**Comment by Ethanout** [comment]
I accept this change of license for the entire Bookshelf project.

**Comment by ascpial** [comment]
I accept this change of license for the entire Bookshelf project.

**Comment by Aeris1One** [comment]
I can't remember what I contributed to Bookshelf but anyway I accept this change of license.

**Reply by theogiraudet** [reply]
Technically, only the Github actions :D

**Comment by KubbyDev** [comment]
I accept this change of license for the entire Bookshelf project.

**Comment by Majoras16** [comment]
I accept this change of license for the entire Bookshelf project.

**Comment by VForiel** [comment]
I accept this change of license for the entire Bookshelf project.

**Comment by theogiraudet** [comment]
Thank you all! The new licence is approved, I will merge that. :)
